packages/react-composites/src/composites/localization/locales/en-US/strings.json

```diff
@@ -23,6 +23,7 @@
     "copyInviteLinkActionedAriaLabel": "Invite link copied",
     "defaultPlaceHolder": "Select an option",
     "dismissSidePaneButtonLabel": "Close",
+    "sidePaneActiveTabAriaLabel": "{header} currently selected",
```

Comment:
'People currently selected' and 'Chat currently selected' might be a bit confusing and might be mistranslated. Can we change this "'{header}' tab is currently selected'" unless Alex P specifically wanted this string.

Reply:
Changed to use aria-selected instead.